ffi/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 #![allow(clippy::unnecessary_box_returns)] // Diplomat requires returning Boxed types
 #![allow(clippy::should_implement_trait)] // Implementing extra traits is not useful for FFI
+#![allow(clippy::needless_lifetimes)]
```

Comment:
nitpick: Add a comment to explain why the lint is disabled.

Reply:
added